packages/zend-locale/library/Zend/Locale/Math.php

```diff
@@ -221,7 +221,7 @@ public static function localize($value)
      */
     public static function exponent($value, $scale = null)
     {
-        $scale = $scale > 0 ? $scale : 0;
+        $scale = max(0, $scale);
```

Comment:
this obviously was for ALL similar changes you made. I would be easier to follow it if you made atomic commits, i.e each of the change as a separate commit. that you could git show and git fixup

highly recommended links to read and follow:
- https://www.freshconsulting.com/atomic-commits/
- https://github.com/keis/git-fixup
- https://github.com/MitMaro/git-interactive-rebase-tool

Reply:
Agreed on the atomic commits. Pure laziness on my side.

Sorry, I have little experience with rebase workflows. Do I understand you correctly that I should rework the fat commit into separate ones via rebasing?

Reply:
I would say yes, you should do that, but I'm not able to enforce that on you, and it's definitely not a merge blocker :), besides @falkenhawk has the final say.

I can't give exact help on how to do that, so I've come up with a quick search:
- https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits
- https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_splitting_a_commit

however, I wanted to say here that the comment to use `max` was intended to all similar changes you made `($foo > 0 ? $foo : 0)`, if you don't plan to update all of them, remove the single `max` usage, so the code would be at least consistent.

Reply:
Totally agreed on the `max`. I'll look into reworking that history. If it looks like a Möbius strip afterwards I'm not taking any responsibility ;)